### ISSUE TITLE
Training on other noise dataset gives resuts worse than unprocessed noisy

### ISSUE THREAD

**Issue by archiki** [anicolson/DeepXi#41]
Hi @anicolson!
I have been working with a custom noise dataset compile from freesound.org which has 7 types of noises- including babble, traffic, etc. I have 10 noise clips for each noise type for training and 8 different clips for testing. I'm using the clean speech from the Librispeech Dataset and the respective train, val and test folders. I trained the network for 200 epochs after deleting the data directory and making the corresponding /set folder as required. Yet I notice that the all the metrics are lower for the trained model as compared to unprocessed speech. ( Initially, I had not deleted the data folder before training, but even after doing that and retraining the results did not improve at all). Here are the metrics:
Model,MOS-LQO,PESQ,STOI,eSTOI
Unprocessed,1.29,1.80,80.68,60.66
DeepXi,1.34,1.58,40.76,25.35
The unprocessed has been calculated in the same way as the DeepXi, by bypassing the model. Can you help me out with what has gone wrong?
Thanks in advance!

**Comment by anicolson** [comment]
Ok,

First, let us look at your training and validation error curves, as well as your training examples. Do you have access to MATLAB?  If so, please do the following:

Please look at log/monitor_epoch.m to plot your training and validation error curves.

Next, in `run.sh` set `--eval_example` to 1 and use TRAIN=1. It will save a mini-batch of training examples before the training starts. Next, use `eval_example.m` to look at your training examples for a mini-batch to see if they look correct.

If you would like to attach images of these to the chat, I can help to see if they make sense.

Please feel free to ask more questions,
Aaron.

**Comment by archiki** [comment]
Hi @anicolson ,
The figures look fine to me. I am attaching them for your reference
![training_epochs](https://user-images.githubusercontent.com/34877432/84266089-8b820780-ab41-11ea-8a09-2116fff16571.png)
![training_batch](https://user-images.githubusercontent.com/34877432/84266267-d56aed80-ab41-11ea-8843-554dfbc60dfa.png)
![training_batch2](https://user-images.githubusercontent.com/34877432/84266281-dbf96500-ab41-11ea-81b7-077e1b8f9f37.png)
![training_batch4](https://user-images.githubusercontent.com/34877432/84266295-e156af80-ab41-11ea-8fc4-1dbc0105402b.png)
![stats](https://user-images.githubusercontent.com/34877432/84266329-ed427180-ab41-11ea-8978-4fd8e8576c6f.png)
Thanks,

**Comment by anicolson** [comment]
Have you tried using the Deep Xi training set to see if it trains normally? It can be used as a reference.

70 noise files is a bit small for the train-clean-100 set which has around 26000 clean speech recordings.

What noise files are you using for validation?

Regards,
Aaron.

**Comment by archiki** [comment]
Hey @anicolson !
I see your point, however, I need to stick with this dataset at least for evaluation purposes. Can you tell me how I can load you pretrained model with my test set? What else do I need apart form the model file, specially in the `/data` folder.
Thanks,

**Comment by anicolson** [comment]
Hi,

All you need is the `data/stats.npz` file. 

Regards,
Aaron.
